Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -390,7 +390,7 @@ void WebProcessProxy::setIsInProcessCache(bool value, WillShutDown willShutDown)
     if (willShutDown == WillShutDown::Yes)
         return;
 
-    send(Messages::WebProcess::SetIsInProcessCache(m_isInProcessCache), 0);
+    sendWithAsyncReply(Messages::WebProcess::SetIsInProcessCache(m_isInProcessCache), []() { });
```

Comment:
Maybe add a comment to explain that we're using sendWithAsyncReply() to make sure the destination process gets woken up long enough to process the IPC.